config/gateway/gateway-plugin/gateway-plugin.yaml

```diff
@@ -3,6 +3,12 @@ kind: Service
 metadata:
   name: gateway-plugins
   namespace: aibrix-system
+  labels:
+    app: gateway-plugins
+    prometheus.io/scrape: "true"
```

Comment:
what's your `kubernetes_sd_configs` setting? I think prometheus configuration should be under annotation?